src/Consolonia.GuiCS/Consolonia.GuiCS.csproj

```diff
@@ -1,7 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Core.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
What do we want to apply from there?

Reply:
this is magic build props which makes it so that the GuiCS is included as an assembly in PlatformSupport package, and doesn't live as it's own top level package.